### ISSUE TITLE
Order NetApp licenses for Kamloops Lab

### ISSUE THREAD

**Issue by stewartshea** [BCDevOps/OpenShift4-RollOut#47]
- [x] SCO for licenses approved
- [x] Order to vendor
- [x] Licenses Received
- [ ] Licenses Applied

**Comment by tosazuwa** [comment]
Waiting on estimates from @DanDeane before the i-store order can be placed

**Comment by DanDeane** [comment]
We had a meeting this morning regarding this.  What further estimates are needed here?  We know how much storage is required.  I will follow up on what paper work would be required for these to move ahead.  @Joe-Craig may have that as he helped with the cluster for the existing 3.11 lab.  @Joe-Craig ?

**Comment by Joe-Craig** [comment]
@tosazuwa  @DanDeane: Are we still looking at 1 HA pair with 2Tb available storage?
If so the SCO for the last ones was "SCO-PROV-156 Amend SCO-PROV-152" so you should just be able to order the same again.

**Comment by tosazuwa** [comment]
@lukegonis , is this sufficient info?

**Comment by jefkel** [comment]
For clarity's sake, @Joe-Craig @DanDeane can you confirm that luke can proceed with ordering:

**3 x "SCO-PROV-156 Amend SCO-PROV-152"**

This covers ONLY the OnTap licenses for the Release 1 implementation (see cluster worksheets for sizing detail), and the Virtual Machines and SAN storage is part of another iStore request.

**Comment by tosazuwa** [comment]
@DanDeane @Joe-Craig Please update ticket

**Comment by Joe-Craig** [comment]
I am talking with our finance department about whether this is a new SCO or we can use the existing.
@jefkel As we have not finalized whether production is going to be physical or virtual are we still going to go ahead and order a third ONTAP setup for production at this time?

**Comment by Joe-Craig** [comment]
@jefkel Looks like a new SCO needs to be done for this request. I will keep you posted.

**Comment by jefkel** [comment]
@Joe-Craig after the standup yesterday, I believe we were waiting to hear options from DXC on how best to proceed and still meet the project targets.

**Comment by Joe-Craig** [comment]
@jefkel  I am working with finance to expedite the SCO. Will talk with the storage team about possibly using a temp license to bridge the gap.

**Comment by kelpisland** [comment]
Based on the outage we suffered this weekend regarding CNS we need to accelerate the rollout of NetApp. Short term is to increase Virtual to 20tb and start migration immediately. Then procure physical ASAP.

Let’s treat storage as a separate concern outside of the platform service agreement to avoid unnecessary dependencies. @DanDeane  @Joe-Craig

**Comment by jefkel** [comment]
Blocked (vacations)

**Comment by Joe-Craig** [comment]
SCO-188 Amend SCO-156 NetApps ONTAP Storage - Complete and sent to the contract office.

**Comment by Joe-Craig** [comment]
Assign to Tolu to chase down with contract office..

**Comment by tosazuwa** [comment]
I am currently following up with Leanne Howes (Hosting Services) to figure out who needs to sign the contract

**Comment by Joe-Craig** [comment]
David Ell has asked questions about the quantities in the SCO. I have supplied answers and I believe that addresses David's issues.

**Comment by tosazuwa** [comment]
Also, the SCO-188 Amend SCO-156 NetApps ONTAP Storage from DXC with the contract Office, should be cleared in 2 days (estimating Monday Jan 13th)

**Comment by Joe-Craig** [comment]
Still waiting for the sign off of " SCO-188 Amend SCO-156 NetApps ONTAP Storage"

**Comment by Joe-Craig** [comment]
Provinces contract office have returned the SCO requesting changes which will need to be approved and re-issued. Still unable to move to procurement.

**Comment by Joe-Craig** [comment]
I believe this went back out to the Provinces contract office today, @tosazuwa can you follow up?

**Comment by tosazuwa** [comment]
SCO 188 for the ONTAP Select licenses is now signed off and Bruce (DXC) is taking it to procurement. @Joe-Craig, please provide ETA for delivery

**Comment by Joe-Craig** [comment]
Have reached out internally to try and get an ETA..

**Comment by tosazuwa** [comment]
Closed. Duplicate of #58
